lib/db.js

```diff
@@ -211,8 +220,29 @@ module.exports = (
   }
 
   DB.prototype.sessions = function (uid) {
-    log.trace({ op: 'DB.sessions', uid: uid })
-    return this.pool.get('/account/' + uid + '/sessions')
+    const promises = [
+      this.pool.get('/account/' + uid + '/sessions')
+    ]
+
+    if (this.redis) {
+      promises.push(this.redis.getAsync(uid))
+    }
+
+    log.trace({ op: 'sessions', uid: uid })
+    return P.all(promises)
+      .spread((tokens, redisTokens) => {
+        const redisSessionTokens = redisTokens ? JSON.parse(redisTokens) : []
+        return tokens.map(function (sessionToken) {
+          const redisToken = redisSessionTokens.find(tok => tok.tokenId === sessionToken.tokenId)
+          const toLog = redisToken ? 'redis.session' : 'DB.session'
+          const mergedToken = Object.assign({}, sessionToken, redisToken)
+          log.info({
```

Comment:
Actually, I'm wondering how valuable this log message is. There's going to be quite a lot of them if we're logging for every session in the array each time. I'd suggest getting rid of it, unless there's some specific reason we think we'll need to go looking for it in Kibana one day...?